operator/gitops/argocd/grafana/dashboards/pipeline-service-dashboard.json

```diff
@@ -3810,6 +3810,129 @@
       },
       "id": 79,
       "panels": [
+        {
+          "description": "Watcher work queue depth",
```

Comment:
```suggestion
          "description": "Tekton Chains work queue depth",
```

Reply:
You still have to this one to update @jkhelil

Reply:
fixed